src/Servers/HttpSys/src/NativeInterop/UrlGroup.cs

```diff
@@ -71,7 +71,7 @@ internal void SetProperty(HttpApiTypes.HTTP_SERVER_PROPERTY property, IntPtr inf
 
         internal void RegisterPrefix(string uriPrefix, int contextId)
         {
-            _logger.LogDebug("Listening on prefix: " + uriPrefix);
+            _logger.LogDebug(LoggerEventIds.RegisteringPrefix, "Listening on prefix: {0} " , uriPrefix);
```

Comment:
Good formatting improvement.
```suggestion
            _logger.LogDebug(LoggerEventIds.RegisteringPrefix, "Listening on prefix: {0}" , uriPrefix);
```